amd_openvx_extensions/amd_migraphx/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ project(vx_amd_migraphx)
 
 set(CMAKE_CXX_STANDARD 14)
 
-list (APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/hip ${ROCM_PATH})
+list (APPEND CMAKE_PREFIX_PATH ${ROCM_PATH} ${ROCM_PATH}/hip)
```

Comment:
@arvindcheru is this required or can we just have $(ROCM_PATH} as /hip is deprecated?

Reply:
Thanks for comments, in the proposed changes we were prioritizing (order) the new path. this is same approach used for search path and not removed old path in current version for all related changes. If there are additional thoughts/comments related to this Please let us know.